src/components/entity-panel/types.ts

```diff
@@ -9,14 +9,22 @@ export interface TreeViewItem {
   tactic?: Tactic;
```

Comment:
Maybe declaring this as types.d.ts would make more sense, or...?

Reply:
It causes issues with the loader and would probably need to be fixed in a dedicated issue

Reply:
That's okay then.